community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/fusion/FusionSchemaIndexProvider.java

```diff
@@ -80,29 +78,46 @@ public IndexAccessor getOnlineAccessor( long indexId, IndexDescriptor descriptor
     @Override
     public String getPopulationFailure( long indexId ) throws IllegalStateException
     {
-        String nativeFailure = nativeProvider.getPopulationFailure( indexId );
-        String luceneFailure = luceneProvider.getPopulationFailure( indexId );
-        if ( nativeFailure != null )
+        String nativeFailure = null;
+        try
```

Comment:
`getPopulationFailure` is designed to throw `IllegalStateException` if called if index isn't in `FAILED` state, sensible methinks.